tests/calc/test_thermo.py

```diff
@@ -12,7 +12,7 @@
                         dewpoint_from_relative_humidity, dewpoint_from_specific_humidity,
                         dry_lapse, dry_static_energy, el, equivalent_potential_temperature,
                         exner_function, gradient_richardson_number, isentropic_interpolation,
-                        isentropic_interpolation_as_dataset, lcl, lfc, lifted_index,
+                        isentropic_interpolation_as_dataset, lcl, lfc, lifted_index, k_index,
```

Comment:
Let's keep the list alphabetized:
```suggestion
                        isentropic_interpolation_as_dataset, k_index, lcl, lfc, lifted_index,
```